packages/api/utils/validation.ts

```diff
@@ -137,3 +137,29 @@ export function isClin(object: unknown): object is Clin {
     (item) => item in object
   );
 }
+
+/**
+ * Check whether a given string is a valid CLIN number.
+ * @param str - The string to check
+ * @returns true if the string is a valid CLIN number; false otherwise
+ */
+export function isClinNumber(str: string): boolean {
+  if (str.length !== 4) {
+    return false;
+  }
+  const num: number = parseInt(str);
+  return num >= 1 && num <= 9999;
+}
```

Comment:
We should investigate changing the API spec to just require that this field be a number. That way if something stringy is provided it'll get dropped at the API GW level (since it's a structural/typing issue). We could even specifying a `minimum` and `maximum` attribute there. That way we don't even have to worry about it on the API side.

Reply:
I think the spec is stringy because typical CLIN values are written and understood with leading zeros like `0001`, `0002`, `0003`, etc.

But certainly if the API Gateway can ensure we get 1 - 9999 by changing the API spec, that's potentially better.